__tests__/createActions.ts

```diff
@@ -0,0 +1,102 @@
+import { createStore, combineReducers, Store } from 'redux'
+import { createAggregate, Modeler, createActions } from '../src/index'
+
+describe('createActions', () => {
+
+  // @ Actions
+
+  function tick() {
+    const date = new Date()
+    const month = date.getMonth() + 1
+    const _date = date.getDate()
+    const hour = date.getHours()
+    const minute = date.getMinutes()
+    const second = date.getSeconds()
+    return `${month}/${_date} ${hour}:${minute}:${second}`
+  }
+  const TimerAC = { tick }
+
+  // @ Model
+
+  interface FollowerST {
+    now: string
+  }
+  const FollowerModel: Modeler<FollowerST> = injects => ({
+    now: '',
+    ...injects
+  })
+  const mutations = {}
+  const subscriptions = {
+    tick (state: FollowerST, timeLabel: string) {
```

Comment:
[NOTE]: subscriptions fn name, must be match resource action fn name. actionSrc return type will map to sectond argument.